Makefile

```diff
@@ -76,6 +75,14 @@ verify: build
 	hack/verify-generated-swagger-spec.sh
 .PHONY: verify
 
+# Verify commit comments.
+#
+# Example:
+#   make verify-commits
+verify-commits:
```

Comment:
Should make this more general name so we can stuff whatever else we find that requires knowledge of the Git process in here